yaml-tests/src/main/java/com/apple/foundationdb/relational/yamltests/server/ExternalServer.java

```diff
@@ -66,6 +67,22 @@ public ExternalServer(File serverJar, final int grpcPort, final int httpPort) {
         this.httpPort = httpPort;
     }
 
+    static {
```

Comment:
This may "catch" other unrelated processes. Can we maybe:
- add "java" as the process name
- only turn this on in non-CI environments

Reply:
Restricting it to `java` makes sense. It can't be strict equality, at least locally, it's a long path, and then `/java`.
We don't (currently) have a way to know that we are in CI, but they are all run in docker images, so there shouldn't really be anything running that we didn't start, so I don't feel like we need to take in that change here.